sphinx-docs/screenshot.py

```diff
@@ -17,6 +17,8 @@
 SCREENSHOT_COMMAND_OPTS = " --no-del -v 0 --from-str '%s' --output-dir %s"
 OUTPUT_PATH = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)),"_build","html","_images"))
 
+from django.core.management import call_command
```

Comment:
Sadly, we seem to have inadvertently merged some undesirable changes while rebasing... Just replace this file with the copy from my own branch.